app/services/github/issue_web_hook_service.rb

```diff
@@ -0,0 +1,27 @@
+module Github
+  class IssueWebHookService
+
+    def self.process(action, payload)
+      begin
+        if action == 'opened'
+          Author.create(
+            name: payload[:title],
+            biography: payload[:body],
+            platform_author_id: payload[:number])
+        elsif action == 'edited'
+          Author.find_or_create_by(platform_author_id: payload[:number])
+            .update(
+              name: payload[:title],
+              biography: payload[:body])
+        elsif action == 'closed'
+          Author.find_by(platform_author_id: payload[:number])
+            .destroy
+        else
+          puts "#{action} is not supported yet."
+        end
+      rescue Exception => e
+        puts "Exception raised during processing web hook payload: #{e.inspect}"
```

Comment:
Since you're catching any exceptions here, the webhook will still be accepted and Github will stop retrying it, even though you were unable to process the data it contains. This will lead to data-loss if there's any sort of error here. How would you change this code to make it better handle these errors?